h2o-algos/src/test/java/hex/tree/PrintMojoTreeTest.java

```diff
@@ -112,10 +113,10 @@ public void testMojoCategoricalPrint_limitedLevels() throws IOException {
       }
 
       final String treeDotz = FileUtils.readFileToString(treeOutput);
-      assertTrue(treeDotz.contains("\"SG_0_Node_0\" -> \"SG_0_Node_1\" [fontsize=14, label=\"[NA]\\n2 levels\\n\"]\n" +
-          "\"SG_0_Node_0\" -> \"SG_0_Node_4\" [fontsize=14, label=\"Iris-virginica\\n\"]\n" +
```

Comment:
Changed the way how we are generating those \ because jgrapht dot importer had problem with it.. tested whether graphviz is still able to read them and it is ok also after this change.

Reply:
if the resulting PNG from dot looks the same than fine

Reply:
it looks